coprocess_grpc.go

```diff
@@ -85,13 +85,29 @@ func (d *GRPCDispatcher) Reload() {}
 // HandleMiddlewareCache isn't used by gRPC.
 func (d *GRPCDispatcher) HandleMiddlewareCache(b *apidef.BundleManifest, basePath string) {}
 
+func grpcCallOpts() grpc.DialOption {
+	recvSize := config.Global().CoProcessOptions.GRPCRecvMaxSize
+	sendSize := config.Global().CoProcessOptions.GRPCSendMaxSize
+	if recvSize > 0 && sendSize > 0 {
+		return grpc.WithDefaultCallOptions(
+			grpc.MaxCallRecvMsgSize(recvSize),
```

Comment:
looking at `grpc` package we see that both `MaxRecvMsgSizeCallOption` and `MaxSendMsgSizeCallOption` have comment `// This is an EXPERIMENTAL API.` which is kind of worrying. If we decide to go with this approach then might wanna keep an eye on it.

Reply:
I think that this will really limit the use cases for our users (like the current implementation does). For users that don't want to override these settings, we use the defaults and compatibility remains the same.